index.html

```diff
@@ -10703,7 +10703,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 &lt;h1 id="slideheading">Quarterly Report&lt;/h1&gt;
 &lt;!-- remaining slide contents --&gt;
 &lt;/div&gt;</pre>
-				<p>In the previous examples, a braille screen reader user may read "sld Quarterly Report" rather than the more verbose "slide Quarterly Report."</p>
+				<p>In the previous examples, a braille screen reader user can read "sld Quarterly Report" rather than the more verbose "slide Quarterly Report."</p>
```

Comment:
```suggestion
				<p>In the previous examples, a braille screen reader user would read "sld Quarterly Report" rather than the more verbose "slide Quarterly Report."</p>
```